GoogleForCreators/web-stories-wp
# Design Panel: Color presets redesign

## Summary
Redesigns the saved colors/styles panel based on the new design.


## Relevant Technical Choices



## To-do

- [x] Review the design system files again to see if anything can be used from there instead.



## User-facing changes
Color Presets panel has been updated to match the style in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=1270%3A14117) Note that the panel header arrow is still per old design since this influences all panels.


**Before:**
![Screenshot 2021-01-13 at 13 13 00](https://user-images.githubusercontent.com/3294597/104445039-ecbaff00-5598-11eb-875c-9f658b5d71cb.png)
![Screenshot 2021-01-13 at 13 13 06](https://user-images.githubusercontent.com/3294597/104445036-ec226880-5598-11eb-8e6e-dcf787f01e20.png)

**After:**
![Screenshot 2021-01-13 at 13 14 05](https://user-images.githubusercontent.com/3294597/104445031-ea58a500-5598-11eb-9e76-32c4fec14aa1.png)
![Screenshot 2021-01-13 at 13 12 24](https://user-images.githubusercontent.com/3294597/104445041-ecbaff00-5598-11eb-9d5d-c3d7262bbeb4.png)
![Screenshot 2021-01-13 at 13 11 38](https://user-images.githubusercontent.com/3294597/104445045-ed539580-5598-11eb-9583-9be0792b98be.png)


## Testing Instructions

- Verify that the saved colors and saved styles panels look similar to [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=1270%3A14117) in case of no presets and in case of added presets; edit mode.
- Note that the headers of the panels will be changed later and can differ.
- Note that currently we only have global colors and this means that the text for Saved color is for globals only and doesn't have a separate button for adding local color. This will be done in #3634 
- Verify that color/style presets can be added and removed as before.


---



Fixes #2908
